src/bots/ama/subcommands/addEvent.js

```diff
@@ -0,0 +1,43 @@
+import client from '../../../client';
+import Event from '../models/Event';
+
+const addEvent = {
+  usage: 'Specify date then title of event',
+  example: 'add 01/01/2020 Celebrate the best year to date',
+
+  handle: (args) => {
+    if (args.length < 2) {
+      client.message.channel.send(
+        'Need to supply date (yyyy-mm-dd) and title of event\n' +
+        '_Example_: 2020-01-01 start of the greatest year ever');
+      return;
+    }
+
+    let date = new Date(args[0] + 'T12:00:00Z');
+
+    if (date.toString() === 'Invalid Date') {
+      client.message.channel.send('Invalid date provided. Must be in format yyy-mm-dd');
+      return;
+    }
+
+    let newEvent = Event({
+      title: args.slice(1).join(' '),
+      date: date
+    }).save();
+
+    newEvent.then(e => client.message.channel.send('Following event has been created:\n' + getFormattedEvent(e)));
```

Comment:
move strings out to a strings constant file (?)

Reply:
would these be constants? They all depend on events.

Reply:
yes the strings themselves should be constants that you import and put in your message.sends, look at the way salary bot strings are done or the stop bot ones

https://github.com/brendacs/stop-bot/blob/master/src/constants.js#L35